### PR TITLE
Mimekitty remove

### PR DESCRIPTION
**Pull request by Groudonmaster**
Refer to issue #2009 

### Intent of your Pull Request

Removes Mimekitty at Shadow's request. Replaces it with the honkbot because it's better and I miss it.

#### Changelog

:cl:
rscdel: Mimekitty has been slaughtered for breaking it's vow of silence
rscadd: To replace the late Mimekitty we're bluespacing in a honkbot. Because they're better anyways.
/:cl:

### CODE REVIEW

**Comment by ShadowDeath6** [comment]
woah!!!!!!!

it's a player reference also so

File: code/modules/mob/living/simple_animal/friendly/cat.dm [hunk 1]
```diff
@@ -262,16 +262,6 @@
...
-/mob/living/simple_animal/pet/cat/mimekitty
-	name = "\improper Mimekitty"
-	desc = "The rare and powerful Mimekitty."
-	icon_state = "mimekitty"
-	icon_living = "mimekitty"
-	icon_dead = "mimekitty_dead"
-	gender = FEMALE
-	var/following = 0
-
 /mob/living/simple_animal/pet/cat/mimekitty/attack_hand(mob/living/carbon/human/M)
```
Comment (Time-Green) [inline code/modules/mob/living/simple_animal/friendly/cat.dm]: Either remove all or nothing.
This probably wont compile if you dont delete this proc

Comment (Groudonmaster) [reply code/modules/mob/living/simple_animal/friendly/cat.dm]: oshit I missed that

Comment (Groudonmaster) [reply code/modules/mob/living/simple_animal/friendly/cat.dm]: Fixed

**Comment by Time-Green** [comment]
Ok so, it's probably the boxstation map causing it, since github still compiles it for some reason.

**Comment by Groudonmaster** [comment]
Broke how fix

**Comment by Time-Green** [comment]
There's multiple maps bla bla
The boxstation map, called tgstation I think, was copied and renamed to yogstation
The old tgstation map is still in the files and probably still has the mimekitty.
Open the tgstation dmm and remove mimekitty from there

**Comment by LarissaMayrink** [comment]
aw god no, don't let it happen

**Comment by Kmc2000** [comment]
mimekitty was bad, player references are bad grr

**Comment by Groudonmaster** [comment]
Just gonna start over with this one. Be back soon.

**Pull request closed without merging**